services/webnn/ort/ort_model_builder.cc

```diff
@@ -101,7 +101,7 @@ void OrtModelBuilder::AddOutput(std::string_view name,
   void* ort_tensor_raw_data = nullptr;
   RETURN_STATUS_IF_FAILED(
       GetOrtApi()->GetTensorMutableData(initializer, &ort_tensor_raw_data));
-  CHECK(ort_tensor_raw_data);
+  // ort_tensor_raw_data can be nullprt when data is empty.
```

Comment:
What's the case?

Reply:
Sometimes we want to create an empty initializer as the `new_shape` of reshape.